src/views/evaluations/evaluations.js

```diff
@@ -5,102 +5,100 @@ function EvaluationsPage(options) {
     ];
     const rows = [
         {
-        name: 'Popescu Adrian',
-        technology: 'Javascript',
-        level: 'Mid 2'
-    },
-    {
-        name: 'Dragan Roxana',
-        technology: 'PHP',
-        level: 'Mid 1'
-    },
-    {
-        name: 'Florescu Mihai',
-        technology: 'Javascript',
-        level: 'Junior 3'
-    },
-    {
-        name: 'Ghoreghe Andrei',
-        technology: 'Ruby',
-        level: 'Mid 2'
-    }
-];
+            name: 'Popescu Adrian',
+            technology: 'Javascript',
+            level: 'Mid 2'
+        },
+        {
+            name: 'Dragan Roxana',
+            technology: 'PHP',
+            level: 'Mid 1'
+        },
+        {
+            name: 'Florescu Mihai',
+            technology: 'Javascript',
+            level: 'Junior 3'
+        },
+        {
+            name: 'Ghoreghe Andrei',
+            technology: 'Ruby',
+            level: 'Mid 2'
+        }
+    ];
+    
     return `
-    ${NAV()}
-    ${EvaluationsTable({
-        items: rows,
-        itemHeadings: headings
-    })}
-    ${Footer()}
-    `
-    ;
+        ${NAV()}
+        ${EvaluationsTable({
+            items: rows,
+            itemHeadings: headings
+        })}
+        ${Footer()}
+    `;
 }
-    
+
 function EvaluationsTableHeader(options) {
-        const headings = [];
+    const headings = [];
 
-        options.headings.forEach(function(el) {
-            const generatedEl = `
+    options.headings.forEach(function (el) {
```

Comment:
There are other, better suited, options for iterating over an array's items and mapping them into another set of information. Any idea what those might be?